### PR TITLE
The Great Multihead PR

### PR DESCRIPTION
**Pull request by laserswald**
This isn't done, but I do have one of the programs done. When it IS done, you can use these tiny tools to get info about your multimonitor displays.

### CODE REVIEW

File: randr/lsm.c [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+#include <err.h>
+
+#include "randr.h"
+#include "../util.h"
+
+static xcb_connection_t *conn;
+static xcb_screen_t *scrn;
+
+static void 
+usage(char *name)
```
Comment (dcat) [inline randr/lsm.c]: might wanna exit after this

**Comment by z3bra** [comment]
Doesn't work for me at all (VMware setup with two monitors):

```
$ ./lsm
$
$ xrandr | grep '\<connected\>'
Virtual1 connected 1920x1200+1680+0 (normal left inverted right x axis y axis) 0mm x 0mm
Virtual2 connected 1680x1050+0+0 (normal left inverted right x axis y axis) 0mm x 0mm
$
```

**Comment by laserswald** [comment]
Huh. I'll check it out. It works fine on my system, but that may just be my
system.

**Comment by laserswald** [comment]
@z3bratabs Can you see if the xcb randr libraries are on your virtual machine?

**Comment by z3bra** [comment]
Judging by the fact `xrandr` work as expected, yes ;)

**Comment by laserswald** [comment]
:C I don't know what's going on.

**Comment by laserswald** [comment]
In order for me to fix this, I need more information and steps to reproduce the bug.

**Comment by z3bra** [comment]
I have a vmware install of archlinux. Grab the latest libxrandr, plug the two monitors and use the "cycle monitors" button to use them in the VM

**Pull request closed without merging**